src/OlStyler.js

```diff
@@ -132,12 +178,13 @@ function lineStyle(linesymbolizer) {
  */
 function createCachedImageStyle(imageUrl, size) {
   const { image, width, height } = imageCache[imageUrl];
+  const maxSide = Math.max(width, height);
   return new Style({
     image: new Icon({
       img: image,
       imgSize: [width, height],
       // According to SLD spec, if size is given, image height should equal the given size.
-      scale: size / height || 1,
+      scale: size / maxSide || 1,
```

Comment:
Please see earlier comment about scaling with height, not with max side.

Reply:
i changed it back